Lib/asyncio/queues.py

```diff
@@ -18,6 +25,16 @@ class QueueFull(Exception):
     pass
 
 
+class QueueShutDown(Exception):
+    """Raised when putting on to or getting from a shut-down Queue."""
+    pass
+
+
+_queue_alive = "alive"
+_queue_shutdown = "shutdown"
+_queue_shutdown_immediate = "shutdown-immediate"
```

Comment:
I suggest a private Enum rather these 3 global variables as:
```
class _QueueState(enum.Enum):
    ALIVE = "alive"
    SHUTDOWN = "shutdown"
    SHUTDOWN = "shutdown_immediate"
```

Reply:
This will introduce a new import of `enum` into `queue` (`asyncio.queues` and `multiprocessing.queues` already transitively import `enum`). Is this fine?

Reply:
I feel it's unnecessary.